src/reactapp/src/api/paypalExpress/createCustomerToken/createCustomerToken.js

```diff
@@ -0,0 +1,12 @@
+import { CREATE_PAYPAL_EXPRESS_TOKEN } from './mutation';
+import sendRequest from '../../sendRequest';
+import LocalStorage from '../../../utils/localStorage';
+import modifier from './modifier';
+
+export default async function createCustomerToken({ returnUrl, cancelUrl }) {
```

Comment:
We are not doing anything with the parameters inside the function. So destructuring them is unnecessary here. a variable like `actionUrls` or `urlInfo` would suffice. Then below `const variables = { ...actionUrls, cartId }`.

Reply:
Got it, changed!